bin/build-chrome-layer.sh

```diff
@@ -6,4 +6,5 @@
 set -e
 
 mkdir -p chromelayer/nodejs/node_modules/
-cd chromelayer/nodejs && npm install chrome-aws-lambda@10.1.0 puppeteer-core@10.1.0 --no-bin-links --no-optional --no-package-lock --no-save --no-shrinkwrap && cd -
+#cd chromelayer/nodejs && npm install chrome-aws-lambda@10.1.0 puppeteer-core@10.1.0 --no-bin-links --no-optional --no-package-lock --no-save --no-shrinkwrap && cd -
+cd chromelayer/nodejs && npm install chrome-aws-lambda@10.1.0 puppeteer-core@10.1.0 --no-bin-links --no-package-lock --no-save --no-shrinkwrap && cd -
```

Comment:
Let's get rid of the commented command - I'm interested to know why you changed this? :)

Reply:
Reverted the change as the `--no-optional` option here was not the cause of my local env issue. See image below.

![Image (1)](https://github.com/aws-samples/aws-step-functions-kendra-web-crawler-search-engine/assets/10222/a1fed9d0-58de-4b64-95b5-da9424eec025)